packages/design-tokens-schema/src/basis-tokens.ts

```diff
@@ -189,3 +223,34 @@ export const ThemeSchema = z
   });
 
 export type Theme = z.infer<typeof ThemeSchema>;
+
+export const addContrastExtensions = (rootConfig: Theme) => {
+  walkColors(rootConfig, (color, path) => {
+    const lastPath = path.at(-1)! as ColorNameKey;
```

Comment:
This is a path fragment right? Could we use a type guard for this that might check fi the colorname exists?